README.md

```diff
@@ -1,2 +1,6 @@
 # openmcyclus
-Repository of Cyclus archetypes to couple Cyclus with OpenMc
+Repository of Cyclus archetypes to couple Cyclus with OpenMC
+
+## Installation 
+To install this archetype library run ``python setup.py install``. Testing 
+to be added soon.
```

Comment:
This is the old way of doing installs. I think now, you'd run `pip install .` from the base directory, but maybe you have to do it this way because Cyclus is so old?

Reply:
I don't know if that will work. I created the `setup.py` file based on the documentation at https://fuelcycle.org/arche/tutorial_py/setup.html, and what I have seen done for other python-based archetype libraries (like d3ploy). I can give this a try if you think it's worth it.

Reply:
I think it is. @munkm or @samgdotson probably know more about this though.

Reply:
Both ways are valid. Though there are some differences. In general, I prefer `pip install .` However, if you never plan on publishing this on PyPI then using `python setup.py install` works. Additionally, one of the key benefits of pip is the automatic installation of dependencies. With `cyclus` being the key dependency, I don't think that feature helps all that much.

Here's a StackOverflow discussion https://stackoverflow.com/questions/15724093/difference-between-python-setup-py-install-and-pip-install

Reply:
I don't plan on publishing this to PyPI. I know that Cyclus is available through conda, but I've never tried it. This archetype will also have OpenMC as a dependency, which I'm not sure if pip would install that.